tests/tasks/remove_and_assert_package_absent.yml

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: "Remove the package '{{ package }}'"
+  package:
+    name: "{{ package }}"
+    state: absent
+
+- name: "Get the rpm package facts"
+  package_facts:
+    manager: "auto"
+
+- name: "Assert removed package '{{ package }}'"
+  assert:
+    that:
+      - package not in ansible_facts.packages
+    msg: "'{{ package }}' is not removed"
```

Comment:
IMHO it is fine to trust that the package tasks will remove the package and the assert is not necessary.